data-access/Queries/Artefact.cs

```diff
@@ -55,16 +60,20 @@ public static string GetQuery(uint? userId, bool mask = false, bool ordered = tr
     public static class FindArtefactComponentId
     {
         private const string _getQuery = @"
-            SELECT $Table_id
+            SELECT DISTINCT $Table_id
             FROM $Table
             JOIN $Table_owner USING($Table_id)
-            WHERE $Table.artefact_id = @ArtefactId
+            WHERE $Where
                 AND $Table_owner.edition_id = @EditionId
             ";
 
-        public static string GetQuery(string table)
+        private const string _normalArt = "$Table.artefact_id = @ArtefactId";
```

Comment:
A couple of comments here would be very helpful - explaining what the different where clauses do.